static/stylesheets/docs-overrides.css

```diff
@@ -0,0 +1,3 @@
ex-shrink:0;padding-right:5px;padding-left:5px;flex-basis:auto;width:100%;max-width:100%}@media (min-width:768px){.su-footer-right{padding-right:20px;padding-left:20px}}.su-footer-right h3{font-family:Work Sans;font-weight:600;font-size:18px;line-height:1.1;color:#2c3458;margin:0 0 14px}.su-footer-right p{margin-bottom:15px;font-family:Work Sans;font-weight:400;font-size:14px;line-height:1.42;color:#2c3458}.su-footer-right-social li{display:inline-block;margin:0 26px 0 0}.su-footer-right-social li img{max-width:24px}@media (min-width:992px){.su-footer-right{box-sizing:border-box;flex-grow:0;flex-shrink:0;padding-right:5px;padding-left:5px;flex-basis:auto;width:33.33333333%;max-width:33.33333333%;margin-left:16.66666667%}}@media (min-width:992px) and (min-width:768px){.su-footer-right{padding-right:20px;padding-left:20px}}.su-footer-right-info{margin-top:63px}@media (min-width:992px){.su-footer-right-info{margin-top:65px}}.su-footer-right-info h3{margin-bottom:24px}.su-footer-bottom{text-align:center;padding:16px 0 11px;border-top:1px solid rgba(44,52,88,.25);font-family:Work Sans;font-weight:400;font-size:14px;line-height:1.42;color:#2c3458}.su-footer-input-group{display:flex;margin-bottom:50px}@media (min-width:992px){.su-footer-input-group{margin-bottom:0}}.su-footer-input-group .su-button{font-size:18px;line-height:1.1;box-shadow:none;width:110px;margin-left:-110px;border-radius:.1875rem;display:inline-block;text-align:center;transition:background-color .24s cubic-bezier(.2,.3,.25,.9)}.su-footer-input-group .su-input{width:inherit;padding:15px 110px 15px 10px;height:55px;border-radius:.1875rem;padding:.75rem 1.25rem;font-family:Work Sans;font-weight:400;font-size:16px;line-height:1.375;color:#2c3458}.su-footer-input-group .su-input:active,.su-footer-input-group .su-input:focus{box-shadow:0 0 3px #2c3458}.su-footer .footer-symbol{left:50%;max-width:20px;position:absolute;top:-10px}.su-footer-input-group{display:block;width:100%;right:0}.su-footer-input-group .su-button{box-sizing:border-box;flex-grow:0;flex-shrink:0;padding-right:.0625rem;padding-left:.0625rem;flex-basis:auto;width:33.33333333%;max-width:33.33333333%;padding:17px 0;margin-left:-25px;font-family:Work Sans;font-weight:600;font-size:16px;line-height:1.375;color:#fff;line-height:normal;background-color:#2c3458;border:1px solid #5f6065;box-shadow:0 8px 10px 0 rgba(0,0,0,.15);border-radius:3px;position:absolute;top:0;right:0}.su-footer-input-group .su-button:hover{box-shadow:none;background-color:#1b2036}.su-footer-input-group .su-input{box-sizing:border-box;flex-grow:0;flex-shrink:0;padding-right:.0625rem;padding-left:.0625rem;flex-basis:auto;width:83.33333333%;max-width:83.33333333%;padding:17px 20px 17px 12px;background:#fff;border:1px solid #5f6065;border-radius:3px}.su-footer-input-group .su-input::placeholder{font-family:Work Sans;font-weight:400;font-size:16px;line-height:1.375;color:#2c3458}.su-footer-input-group:active,.su-footer-input-group:focus{box-shadow:0 0 3px #2c3458}
```

Comment:
🤔 not sure how I feel about the styles being bundled. It will make it more difficult to change in the future.

Also appears there are references to assets that are not present.

Reply:
@jamesdphillips we'll re-upload the styles un-minified. We found one asset that was improperly linked and will pull that out!